lib/elixir_console_web/live/console_live/sidebar_component.html.leex

```diff
@@ -11,7 +11,7 @@
     <% end %>
     </ul>
   </div>
-  <div class="flex-1 flex flex-col justify-end p-2">
+  <div id="documentation-output" class="flex-1 flex flex-col justify-end p-2">
```

Comment:
Not sure if this name is appropriate as this section shows other things that are not documentation. I can't think of a better name though as this shows different things. We could however, add this id or a similar one in one of the spans below, where we display the contextual help, or maybe even wrap those three spans with a div with the id. What do you think?

Reply:
I agree with you. If we go for the second solution (a div wrapping the three spans), is there a possibility that the UI gets changed?

Reply:
I think a div wrapping the three spans gives semantics to that section so I see it as a nice addition, even if we didn't need it for the tests. Also I could see each section being a separate component in the future so in that case we would also need an element that wraps the component.
The UI should not be affected by adding a wrapping div as all padding/margin/borders default to zero.